src/core/io/device/io_controller.cpp

```diff
@@ -73,7 +73,7 @@ void IODeviceController::addHandle(HandleDescriptor &paHandleDescriptor) {
 }
 
 void IODeviceController::fireIndicationEvent(IOObserver* paObserver) {
-  startNewEventChain((CProcessInterface*) paObserver);
+  startNewEventChain((CProcessInterfaceFB*) paObserver);
```

Comment:
can you please change this to a c++ style cast?

Reply:
done